<?xml version="1.0" encoding="utf-8"?><testsuite errors="0" failures="0" name="pytest" skipped="0" tests="3" time="28.992"><testcase classname="test_login.TestLogin" file="test_login.py" line="36" name="test_login_1_failed[data0]" time="9.294"></testcase><testcase classname="test_login.TestLogin" file="test_login.py" line="36" name="test_login_1_failed[data1]" time="10.027"></testcase><testcase classname="test_login.TestLogin" file="test_login.py" line="36" name="test_login_1_failed[data2]" time="9.535"></testcase></testsuite>